modules/message/src/main/ThreadRepo.scala

```diff
@@ -84,5 +87,7 @@ object ThreadRepo {
 
   def visibleByUserQuery(user: String) = $doc("visibleByUserIds" -> user)
 
+  def participantQuery(user: String) = $or($doc("creatorId" -> user), $doc("invitedId" -> user))
```

Comment:
There's no DB indexes on these fields, and even if there were, the `$or` query would be very expensive. We only select on `visibleUserIds` because that's the indexed field.

Reply:
Right. Is it in any way possible to work around that, or can this issue just not be fixed?

Reply:
Could I create a new field `participants` that gets indexed like visibleUserIds but always contains the two participants?